internal/pkg/static/templates/index/index-blazar.html

```diff
@@ -164,6 +164,17 @@
                 });
             });
 
+            $(document).ready(function() {
+              $('div[id="version-alert"]').on('click', function() {
+                const height = $(this).attr("data-height")
+                $('details:has(#registerVersionForm)').prop('open', true);
+                $('#registerVersionForm #height').val(height);
```

Comment:
unrelated to the PR, but we shouldn't have three elements with the same id (`#height`) :joy: